stdlib/public/stubs/Stubs.cpp

```diff
@@ -37,6 +37,9 @@
 #include <cstdlib>
 #include <cstring>
 #if defined(__CYGWIN__) || defined(_MSC_VER)
+#include <vector>
+#include <string>
+#include <assert.h>
```

Comment:
Nitpick: `<cassert>` instead of `<assert.h>`. Also, I'm not sure why these are guarded under Windows.